ios/FluentUI/AvatarGroup/AvatarGroup.swift

```diff
@@ -266,8 +265,8 @@ public struct AvatarGroup: View, TokenizedControlView {
     var displayedAvatarAccessibilityLabels: [String] {
         var labels: [String] = []
         for i in 0..<avatarsToDisplay {
-            let avatar = state.avatars[i]
-            labels.append(avatar.accessibilityLabel ?? avatar.primaryText ?? avatar.secondaryText ?? "")
+            let avatarState = state.avatars[i].state
+            labels.append(avatarState.accessibilityLabel ?? avatarState.primaryText ?? avatarState.secondaryText ?? "")
```

Comment:
```swift
if let label = avatarState.accessibilityLabel ?? avatarState.primaryText ?? avatarState.secondaryText {
labels.append(label)
}
```

Is there a need to append a empty string?

Reply:
I agree--not a fan of having that blank in the array of accessibility items.